src/app/auth/login/login.component.scss

```diff
@@ -0,0 +1,29 @@
+.login_container {
+    width: 300px;
+    margin: 100px auto;
+
+    .login_container__name {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        font-size: 25px;
+        margin-bottom: 20px;
+        color: rgb(110, 115, 119);
+    }
+
+    mat-form-field {
```

Comment:
nit: not smth we need to fix in this pr, but try to avoid element's styling, uses classes instead